### PR TITLE
Devise attributes update

### PR DESCRIPTION
**Pull request by D-Zaleski**
added attributes such as "address" to user model

### CODE REVIEW

File: app/views/devise/registrations/new.html.erb [hunk 1]
```diff
@@ -1,25 +1,213 @@
...
-  </div>
-<% end %>
+  <%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name), data: { turbo: :false }) do |f| %>
+    <%= f.error_notification %>
+
+    <div class="form-inputs">
+      <%= f.input :first_name,
+                  required: true,
+                  autofocus: true,
+                  input_html: { autocomplete: "given-name" } %>
```
Comment (akramerin) [inline app/views/devise/registrations/new.html.erb:11]: I have never seen an autocomplete for a name before. is this necessary? Feel like there could be some weird results.

File: db/migrate/20230307161705_add_address_to_user.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class AddAddressToUser < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :address, :string
+    add_column :users, :country, :string
+    add_column :users, :city, :string
+    add_column :users, :post_code, :string
```
Comment (akramerin) [inline db/migrate/20230307161705_add_address_to_user.rb:6]: is post_code not usually an integer?

Comment (akramerin) [reply db/migrate/20230307161705_add_address_to_user.rb:6]: (but maybe this is a german thing)

**Review by akramerin — COMMENTED** [review]
Have some tiny questions, the rest (to be really github-style): lgtm

File: db/migrate/20230307161705_add_address_to_user.rb (hunk 2 above)
Comment (D-Zaleski) [reply db/migrate/20230307161705_add_address_to_user.rb:6]: england has letters

Comment (akramerin) [reply db/migrate/20230307161705_add_address_to_user.rb:6]: i guess it is a GRAY area (see what i did there )

fine by me then

File: app/views/devise/registrations/new.html.erb (hunk 1 above)
Comment (D-Zaleski) [reply app/views/devise/registrations/new.html.erb:11]: it's fine. If issues arise I will deal with it.

Comment (akramerin) [reply app/views/devise/registrations/new.html.erb:11]: allright

**Review by syboulette — APPROVED** [review]

**Review by akramerin — APPROVED** [review]

**Pull request merged**